GitHubMainApplication/sortOperation.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GitHubMainApplication
+{
+    public class SortOperation
+    {
+        public SortOperation(int[] array) {
+            if (array.Length != 0)
```

Comment:
change the if condition to count!=0, 
also add another parameter to the function that reads the count from the main class

Reply:
ill update it with count variable once main method is compleated